frontend/src/utils/formatters.ts

```diff
@@ -57,7 +57,7 @@ export function unformatServiceArea(state: State, formattedServiceArea: string):
 function parseFormattedServiceArea(formattedServiceArea: string) {
   let [c, z] = formattedServiceArea.split(' / ')
   return {
-    county: snakeCase(c),
+    county: snakeCase(c.toLowerCase()),
```

Comment:
Please define a new function `snakeCase` in https://github.com/bayesimpact/tds/blob/master/frontend/src/utils/string.ts and use it here instead.